### PR TITLE
Add MAINTAINERS file

### PR DESCRIPTION
**Pull request by tianon**
cc @ehazlett

### CODE REVIEW

**Comment by tianon** [comment]
@ehazlett I also added an initial `CONTRIBUTING.md` here, so if you'd be willing to take a look, I'd appreciate it :pray:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Contributing to boot2docker
+
+[![Build Status](https://travis-ci.org/docker/boot2docker.svg)](https://travis-ci.org/docker/boot2docker)
+
+## Be Respectful
+
+If you're not a maintainer of the boot2docker project, please try to refrain from making authoritative comments (especially misleading `LGTM`).  On the flip side, constructive comments contributing to useful dialog are welcome and encouraged.
+
+## Commit Messages
```
Comment (ehazlett) [inline CONTRIBUTING.md:9]: Maybe a recommendation instead of hard force here.

**Comment by ehazlett** [comment]
Other than the comment, looks good!

File: CONTRIBUTING.md (hunk 1 above)
Comment (tianon) [inline CONTRIBUTING.md:9]: Yeah, good call -- too hard to police. :+1:

Updated!

**Comment by ehazlett** [comment]
LGTM

**Comment by tianon** [comment]
:heart:

**Comment by ehazlett** [comment]
yay :fireworks:

**Pull request merged**